src/modules/dataGrid.scss

```diff
@@ -26,18 +26,25 @@ $blackAlpha: rgba(0, 0, 0, .5);
         }
         &_table_component {
             .table-cell:last-child {
-                padding-right: inherit;
+                padding-right: 11px !important;
```

Comment:
I imagine `!important` is in fact important? ;-)

Reply:
Do we need a SASS Lint comment?

Reply:
Added.

Reply:
Yeah, they need to be important because of the old school way of class names taking priority.

Reply:
Stinkin' legacy CSS.  Got it. 😁